web/src/repogroups/RepogroupPage.tsx

```diff
@@ -0,0 +1,288 @@
+                        <h2 className="font-weight-normal">
+                            <SourceRepositoryMultipleIcon className="icon-inline mr-2" />
+                            Repositories
+                        </h2>
+                        <p>
+                            Using the syntax{' '}
+                            <span className="text-monospace">
+                                <span className="repogroup-page__keyword-text">repogroup:</span>
+                                {props.repogroupMetadata.name}
+                            </span>{' '}
+                            in a query will search these repositories:
+                        </p>
+                        <div className="repogroup-page__repo-list row">
+                            <div className="col-lg-6">
+                                {props.repogroupMetadata.repositories
+                                    .slice(0, Math.ceil(props.repogroupMetadata.repositories.length / 2))
+                                    .map(repo => (
+                                        <RepoLink key={repo.name} repo={repo} />
+                                    ))}
+                            </div>
+                            <div className="col-lg-6">
+                                {props.repogroupMetadata.repositories
+                                    .slice(
+                                        Math.ceil(props.repogroupMetadata.repositories.length / 2),
+                                        props.repogroupMetadata.repositories.length
+                                    )
+                                    .map(repo => (
+                                        <RepoLink key={repo.name} repo={repo} />
+                                    ))}
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    )
+}
+
+const RepoLink: React.FunctionComponent<{ repo: RepositoryType }> = props => (
+    <li className="repogroup-page__repo-item list-unstyled mb-3" key={props.repo.name}>
+        <a href={`https://${props.repo.name}`} target="_blank" rel="noopener noreferrer">
+            {props.repo.codehost === CodeHosts.GITHUB && (
+                <GithubIcon className="icon-inline repogroup-page__repo-list-icon" />
+            )}
+            {props.repo.codehost === CodeHosts.GITLAB && (
+                <GitlabIcon className="icon-inline repogroup-page__repo-list-icon" />
+            )}
+            {props.repo.codehost === CodeHosts.BITBUCKET && (
+                <BitbucketIcon className="icon-inline repogroup-page__repo-list-icon" />
+            )}
+        </a>
+        <Link to={`/${props.repo.name}`} className="text-monospace repogroup-page__web-link">
+            {props.repo.name}
+        </Link>
+    </li>
+)
+
+interface RepogroupPageLogoProps extends ThemeProps, Exclude<React.ImgHTMLAttributes<HTMLImageElement>, 'src'> {
```

Comment:
```suggestion
interface RepogroupPageLogoProps extends Exclude<React.ImgHTMLAttributes<HTMLImageElement>, 'src'> {
```

It doesn't seem to use `ThemeProps`?